target/linux/rtl838x/Makefile

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (C) 2008-2011 OpenWrt.org
```

Comment:
You are not affiliated with OpenWrt, so please remove all the "LEDE" and "OpenWrt.org" copyright notices.